synapsemonitor/__main__.py

```diff
@@ -70,6 +83,28 @@ def build_parser():
     )
     parser_monitor.set_defaults(func=monitor_cli)
 
+    parser_create_view = subparsers.add_parser(
+        'create-file-view',
+        help='Creates a file view that will list all the File entities under '
+             'the specified scopes (Synapse Folders or Projects). This will '
+             'allow you to query for the files contained in your specified '
+             'scopes. This will NOT track the other entities currently: '
+             'PROJECT, TABLE, FOLDER, VIEW, DOCKER.'
+    )
+    parser_create_view.add_argument(
+        'name', metavar='NAME', type=str,
+        help='File View name'
+    )
+    parser_create_view.add_argument(
+        'project_id',
+        help='Synapse Project Id to store file view in'
+    )
+    parser_create_view.add_argument(
+        '--scope_ids', nargs='+', required=True,
+        help='Synapse Folder / Project Ids'
+    )
+    parser_create_view.set_defaults(func=create_file_view_cli)
+
```

Comment:
I'm surprised this doesn't already exist.  This is equivalent to an `ls` inside a scope.  Very cool